config/settings.yml

```diff
@@ -947,9 +944,7 @@ vetext:
 hqva_mobile:
   url: "https://veteran.apps.va.gov"
   mock: false
-  key_path: /app/secret/health-quest.key
```

Comment:
You modified this 6 months ago [here](https://github.com/department-of-veterans-affairs/vets-api/pull/16561/files#diff-e769bbb8c1ba3711c5403b424ed9c218ffafba7f1890ee394717196f28ff4540). But yeah, I don't see it used anywhere.